supabase/tests/36_test_get_weekly_stats_window.sql

```
@@ -0,0 +1,30 @@
+BEGIN;
+
+SELECT plan(2);
+
+DELETE FROM public.daily_version
+WHERE app_id = 'com.test.getweeklystatswindow.app';
+
+-- Inside the 7-day inclusive window: today and 6 days ago.
+INSERT INTO public.daily_version (date, app_id, version_name, get, fail, install, uninstall)
```

SQLFluff (4.2.2)

[error] Line is too long (92 > 80).

(LT05)